jdi-light-angular-tests/src/main/java/io/github/com/pages/sections/AutocompleteSection.java

```diff
@@ -0,0 +1,28 @@
+package io.github.com.pages.sections;
+
+import com.epam.jdi.angular.elements.common.*;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.*;
+
+public class AutocompleteSection {
+    @UI("#mat-input-0")
+    public AutoComplete autocompleteOverview;
+
+    @UI("#mat-slide-toggle-3-input")
+    public SlideToggle autocompleteDisableInput;
+
+    @UI("#mat-input-1")
```

Comment:
Add meaningful ids to elements as described [here](https://epam-my.sharepoint.com/personal/alexey_girin_epam_com/_layouts/15/Doc.aspx?sourcedoc=%7BE3A93BE7-D0FF-45FD-B3D2-804A31B2482C%7D&file=Angular_tests_guide.docx&action=default&mobileredirect=true&cid=f2b07436-ee73-4dad-9ab6-60f8b7b539fc) and use them in tests

Reply:
Done.